src/app/(BottomTabNavigation)/AllCases/QRCodeScanner/index.tsx

```diff
@@ -57,44 +86,64 @@ function QRCodeScannerScreen() {
   const handleBarCodeScanned = async (result: BarCodeScannerResult) => {
```

Comment:
Same with this useEffect above. Move the function declaration outside of the useEffect. 

Also, we should only have one useEffect with an empty dependency array. This will trigger once when the screen renders for the first time, so we should be able to combine the useEffects together.